src/njoy21/input/PLOTR/Card5/Eh.hpp

```diff
@@ -0,0 +1,18 @@
+struct Eh {
+  using Value_t = double;
+  static std::string name(){ return "eh"; }
+  static std::string description(){
+    return
+      "The eh argument specifies the upper bound for the energy (x) axis.\n\n"
+      "Checks that eh has a greater value than el.\n\n"
+      "If using the default value, the default value should be used for el\n"
+      "and xstep as well.";
+  }
+  static Value_t defaultValue( const double el ){
```

Comment:
I'm don't understand what you mean.  el in this file is still the lower energy bound.  The default value is just set to be equal to it because using this default value requires the use of the default values for el and xstep, so it's meaningless and will be changed by the code.  The verify function just checks to be sure that eh is the same as el (default) or greater.  I believe the fortran also checks for a similar relation.

Reply:
Ok, I misunderstood.